packages/core-typings/src/IMessage/IMessage.ts

```diff
@@ -102,14 +104,14 @@ export interface IMessage extends IRocketChatRecord {
 	starred?: { _id: IUser['_id'] }[];
 	pinned?: boolean;
 	unread?: boolean;
-	temp?: boolean;
```

Comment:
Keep it; there are other situations beyond OTR were a message is temporary (e.g. network latency).